homeassistant/components/roborock/coordinator.py

```diff
@@ -59,6 +59,7 @@ def __init__(
 
         if mac := self.roborock_device_info.network_info.mac:
             self.device_info[ATTR_CONNECTIONS] = {(dr.CONNECTION_NETWORK_MAC, mac)}
+        self.maps: dict[int, str] = {}
```

Comment:
Can you make this expose room information as well? (for my options flow)

I think this would mean moving `create_coordinator_maps()` into the coordinator, and calling the map parser there as well.
This also raises another question; what if the user edits the rooms (from the Roborock app) while HA is running?  At the very least, we could document that users should manually reload the integration from Devices after doing that, or we could make `RoborockMap._create_image()` update the room information in the coordinator.

Reply:
I think create_coordinator_maps makes sense in the image file. It instead would call a seperate coordinator function to update the rooms.

Typically, I try to avoid adding entities in the middle of a run, so yes the user should reload. A documentation update would probably be nice. I can try to do it down the line. But this PR does not change the functionality of that, and that is already the case, so I will not tie it in with this PR.